python/pycylon/__init__.py

```diff
@@ -12,6 +12,32 @@
  # limitations under the License.
  ##
 
-from pycylon.data.table import (Table)
-from pycylon.data.csv import csv_reader
-from pycylon.ctx.context import CylonContext
\ No newline at end of file
+from pycylon.data.table import Table
+from pycylon.ctx.context import CylonContext
+import numpy as np
+
+
+int = np.int
+int8 = np.int8
+int16 = np.int16
+int32 = np.int32
+
+uint = np.uint
+uint8 = np.uint8
+uint16 = np.uint16
+uint32 = np.uint32
```

Comment:
is it a good idea to do this?

Reply:
I will remove it. This was just for a test.